master/internal/api_experiment.go

```diff
@@ -1302,7 +1300,7 @@ func (a *apiServer) PatchExperiment(
 				err = runCheckpointGCTask(
 					a.m.system, a.m.rm, a.m.db, taskID, modelExp.JobID, modelExp.StartTime,
 					taskSpec, modelExp.ID, modelExp.Config, checkpoints, []string{fullDeleteGlob}, true,
```

Comment:
delete tensorboard here feels weird to me, I don't know why changing the checkpoint storage would delete the tensorboard files

I know this is preexisting behavior but seems like it is a bug to me

(sorry I told you to flip it back I wasn't thinking just saw the diff and assumed unintentional)

Reply:
Yeah, flipped it back again -- and adding a comment to the test ticket to create a case for this.